catalogue/aarch64-faults/README.md

```diff
@@ -31,3 +31,21 @@ in the folder `${KUT_DIR}`:
     % sh ./litmus/run.sh
 
 > **_NOTE:_** litmus7 does not support MTE and, consequently, cannot run the test LDRredF.
+
+Herd7 support for faults
+------------------------
+
```

Comment:
Is it worth adding the syntax of the fault atom as well? Something like

fault(<Pn>:<label>,<location>,<fault_type>)

And then we explain what <Pn>, <label> ... is for and which fields are optional?